stories/helpers/stories.ts

```diff
@@ -84,3 +85,36 @@ export function cleanupTemplate(template: string): string {
 		.replace(/\n{2,}\t/gm, '')
 		.replace(/ {2,}/gm, ' ');
 }
+
+export function generateInputs(inputs: Record<string, unknown>, argTypes: StrictArgTypes, disableBooleanAttributes = false): string {
+	return Object.entries(inputs).reduce((acc, [name, value]) => {
+		const argType = argTypes[name];
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+		if (argType['table'] && argType['table'].category !== 'inputs') {
+			return acc;
+		}
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+		const defaultValue: unknown = argType['table']?.defaultValue?.summary;
+		if (value === defaultValue || value === null || value === undefined) {
+			return acc;
+		}
+		// Let's treat boolean inputs as booleanAttributes for stories
+		if (!disableBooleanAttributes && typeof value === 'boolean') {
+			if (value) {
+				return `${acc} ${name}`;
+			}
+			return acc;
+		}
```

Comment:
Can you explain this choice? Do we have any way of knowing if it is a transformed input or not?

Reply:
What do you mean by transformed input in this context?

Reply:
Something like `@Input({ transform: someTransformer })`